developer/contributing.md

```diff
@@ -31,6 +31,84 @@ Start small and open up a dialogue with us. This will help to get your contribut
 -   Contributors should review the CLA.
 -   Code reviewed before merge acceptance.
 
+### Pull request template
+
+```
+Resolves #issueNumber
+Impact breaking|minor|bug|style|refactor
+
+## Issue
+
+Description of the issue this PR is solving, why it's happening and how it can be reproduced. This may differ from the original ticket as you now have more information at your disposal.
+
+## Solution
+
+Summarize the your solution to the problem, as well as any things you may have tried along the way that may not have worked.
+
+## Breaking changes
+
+- If you have a breaking changes, list them here, otherwise list none.
```

Comment:
We may need to give some examples (here or linked) as to what constitutes a breaking change. I'm not sure we have this documented somewhere, but it would be valuable.

Reply:
Breaking change example: https://github.com/reactioncommerce/reaction/pull/3331

Reply:
https://github.com/reactioncommerce/reaction/pull/3524